src/app/genome/genome.view.model.ts

```diff
@@ -3,6 +3,7 @@ export default class GenomeViewModel {
     private taxonomy: any[] = [];
     private mainInfoFields: any[] = [
         {"name": "Organism", "value": "name"},
+        {"name": "NCBI Taxonomy id", "value": "taxonomy_id"},
         {"name": "Assembly level", "value": "assembly_level"}, 
         {"name": "Genbank version", "value": "genbank_version", "ftp_path": "ftp_path"}, 
```

Comment:
This line `{"name": "Genbank version", "value": "genbank_version", "ftp_path": "ftp_path"}` also need to be changed to contain RefSeq Version.